### ISSUE TITLE
When recalling a message it gets deleted, until it is sent or postponed again. Can cause loss of messages.

### ISSUE THREAD

**Issue by NilsIrl** [neomutt/neomutt#3103]
## Expected Behaviour

(this isn't exactly a bug per se but it definitely wasn't the behavior I was expecting)

1. Recall a message
2. Make (neo)mutt crash
3. Re-open (neo)mutt
4. Recall message again and start from where you had left of

## Actual Behaviour

Step 4 doesn't work because the message is deleted when it is recalled in step 1. So there is no longer a message to recall.

If NeoMutt crashed, did your OS create a 'coredump' file?

In my case the problem occurred because my computer as a whole crashed.

## NeoMutt Version

```
NeoMutt 20210205
Copyright (C) 1996-2020 Michael R. Elkins and others.
NeoMutt comes with ABSOLUTELY NO WARRANTY; for details type 'neomutt -vv'.
NeoMutt is free software, and you are welcome to redistribute it
under certain conditions; type 'neomutt -vv' for details.

System: Linux 5.10.75 (x86_64)
ncurses: ncurses 6.2.20200212 (compiled with 6.2.20200212)
libidn: 1.36 (compiled with 1.36)
GPGME: 1.15.1-unknown
OpenSSL: OpenSSL 1.1.1l  24 Aug 2021
libnotmuch: 5.3.0
storage: lmdb
compression: zlib

Configure options: --prefix=/nix/store/3x5i862m17inq1la2wlcld4b1y9k6b10-neomutt-20210205 --enable-autocrypt --gpgme --gss --lmdb --notmuch --ssl --sasl --with-homespool=mailbox --with-mailpath= --disable-include-path-in-cflags ac_cv_path_SENDMAIL=sendmail --zlib

Compilation CFLAGS: -g -O2 -std=c99 -fno-delete-null-pointer-checks -D_ALL_SOURCE=1 -D_GNU_SOURCE=1 -D__EXTENSIONS__ -DNCURSES_WIDECHAR

Default options:
  +attach_headers_color +compose_to_sender +compress +cond_date +debug
  +encrypt_to_self +forgotten_attachments +forwref +ifdef +imap +index_color
  +initials +limit_current_thread +multiple_fcc +nested_if +new_mail +nntp +pop
  +progress +quasi_delete +regcomp +reply_with_xorig +sensible_browser +sidebar
  +skip_quoted +smtp +status_color +timeout +tls_sni +trash

Compile options:
  +autocrypt +bkgdset +color +curs_set +fcntl -flock -fmemopen +futimens
  +getaddrinfo -gnutls +gpgme +gss +hcache +homespool +idn +inotify
  -locales_hack -lua +meta -mixmaster +nls +notmuch +openssl +pgp +regex +sasl
  +smime +sqlite +start_color +sun_attachment +typeahead
MAILPATH="mailbox"
PKGDATADIR="/nix/store/3x5i862m17inq1la2wlcld4b1y9k6b10-neomutt-20210205/share/neomutt"
SENDMAIL="/usr/sbin/sendmail"
SYSCONFDIR="/nix/store/3x5i862m17inq1la2wlcld4b1y9k6b10-neomutt-20210205/etc"

To learn more about NeoMutt, visit: https://neomutt.org
If you find a bug in NeoMutt, please raise an issue at:
    https://github.com/neomutt/neomutt/issues
or send an email to: <neomutt-devel@neomutt.org>
```

## Extra Info

* Is your email local (maildir) or remote (IMAP)?

Maildir

TLDR: There's basically nothing that can be done if (neo)mutt crashes when a writing an email, and that's regardless of whether the email was postponed in the first place. It's just that one would expect that if they saved the message once they would be safe.
